src/Middleware/Csrf.php

```diff
@@ -91,6 +93,6 @@ private function getTokenFromRequest(ServerRequestInterface $request): ?string
             $token = \reset($headers);
         }
 
-        return TokenMask::remove($token);
+        return is_string($token) ? TokenMask::remove($token) : null;
```

Comment:
In what case it won't be a string?

Reply:
`reset ` return boolean if the array is empty